EIPS/eip-5269.md

```diff
@@ -0,0 +1,35 @@
+---
+eip: 5269
+title: Human Readable Interface Detection
+description: An interface returning Human Readable Interface
+author: Zainan Victor Zhou (xinbenlv@)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): ERC
+created: 2022-07-15
+requires (*optional): <EIP number(s)>
+---
+
+## Abstract
+An interface returns list of ERC numbers and other numbers in the future, to make it human readable, as opposed to EIP-165 which returns a hashed code of interface in the machine readable manner.
+
+## Motivation
+TODO 
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+TODO 
+
+## Rationale
+TODO 
+
+## Backwards Compatibility
+TODO 
```

Comment:
I removed backward compatibility section, if that's ok

Reply:
No, it is not. All EIPs **must** have a backwards compatibility section, even if it is as simple as the one above.

Reply:
For future reference: I usually allow `TODO` for sections like Backward Compatibility and Security Considerations for Draft EIPs.  These sections need to be filled in before Review, but it can be a helpful reminder to fill them with `TODO` initially so it is more obvious they aren't complete yet.